source/common/router/config_impl.h

```diff
@@ -302,6 +303,12 @@ class RouteEntryImplBase : public RouteEntry,
 
   bool isDirectResponse() const { return direct_response_code_.valid(); }
 
+  bool isRedirect() const {
+    if (!isDirectResponse())
```

Comment:
Nit: Use braces for even single line `if` statements.

Reply:
.. and merge.

Reply:
Sure.